src/testup/minitest_setup.rb

```diff
@@ -18,73 +18,55 @@
 # run the tests when SketchUp exits because Minitest.autoload uses at_exit {}.
 
 require "rubygems"
-gem "minitest"
-require "minitest"
 require "minitest/spec"
 require "minitest/mock"
 
-
-# TestUp modifications of Minitest.
-module Minitest
-
-  class << self
-
-    # In case some tests cause crashes it's nice if the name of the test is
-    # output to the debugger before the test is run.
-    # TODO(thomthom): Review if this can be done without overriding the method.
-    unless method_defined?(:testup_run_one_method)
-      alias :testup_run_one_method :run_one_method
-      def run_one_method(*args)
-        klass, method_name, reporter = args
-        TestUp::Debugger.output("Running: #{klass.name}.#{method_name}")
-        start_time = Time.now
-        result = self.testup_run_one_method(*args)
-        lapsed_time = Time.now - start_time
-        TestUp::Debugger.output("> Elapsed time: #{lapsed_time}s")
-        result
-      end
+module TestUp
+  module MinitestPrepend
+    def run_one_method(klass, method_name)
+      TestUp::Debugger.output("Running: #{klass.name}.#{method_name}")
+      start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
Can't we use the existing timer utility method here?

Reply:
See above re time interval.